blog/_posts/2025-04-24-network-isolated-cluster.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Network Isolated Cluster"
+description: "Use network isolated AKS clusters to restrict outbound traffic at a glance"
+date: 2025-04-24
+author: Charles Wu
+categories: 
+- security
+---
+
+# Network isolated Azure Kubernetes Service (AKS) cluster
+
+By default, standard SKU Azure Kubernetes Service (AKS) clusters have unrestricted outbound internet access. This level of network access allows nodes and services you run to access external resources as needed. For those organizations who have strict security and compliance requirements, to regulate and control egress (outbound) network traffic from a cluster is a must have for the purpose of eliminating data exfiltration.
+
+Previously, if a user wants to restrict outbound traffic for an AKS cluster, firewall is the common option to go with. However, this solution has some shortcomings: 
+1. It could be cumbersome and complicated since you have to figure out the endpoints of the AKS cluster and your own application, which are required for outbound traffic and to create corresponding egress firewall rules. 
```

Comment:
Small rewording here. Suggest changing  "It could be cumbersome and complicated since you have to figure out the endpoints of the AKS cluster and your own application, which are required for outbound traffic and to create corresponding egress firewall rules." to "It could be cumbersome and complicated since you have to figure out the endpoints of the AKS cluster and your own application, which are required for outbound traffic and creating corresponding egress firewall rules."